frontend/apps/pablo/updaters/bonds/Updater.tsx

```diff
@@ -8,22 +8,16 @@ const Updater = () => {
   const { bondOffers, supportedAssets, apollo } = useStore();
 //   const lpRewardingPools = useAllLpTokenRewardingPools();
 
-  const { putBondOffers } = useStore();
+  const { setBondOffers } = useStore();
   const { parachainApi } = useParachainApi(DEFAULT_NETWORK_ID);
 
   useEffect(() => {
     if (parachainApi) {
       fetchBondOffers(parachainApi).then((decodedOffers) => {
-        putBondOffers(decodedOffers);
+        setBondOffers(decodedOffers);
```

Comment:
(question) is this different from `putBondOffers` in `bonds.utils.ts`?

Reply:
i have renamed `putBondOffers` to `setBondOffers`